cabal.project.freeze

```diff
@@ -206,7 +206,7 @@ constraints: any.Boolean ==0.2.4,
              any.optparse-applicative ==0.17.0.0,
              optparse-applicative +process,
              any.pact ==4.4,
-             pact -build-tool +cryptonite-ed25519 +no-advice -with-integer-gmp,
+             pact -build-tool +cryptonite-ed25519 +no-advice -tests-in-lib -with-integer-gmp,

```

Comment:
why was this needed? it's off by default

Reply:
This file is generate by `cabal freeze`, which adds it by default, even if default it off.